core/core-configuration-layer.el

```diff
@@ -2398,11 +2398,12 @@ depends on it."
 
 (defun configuration-layer//package-delete (pkg-name)
   "Delete package with name PKG-NAME."
-  (let ((p (cadr (assq pkg-name package-alist))))
-    ;; add force flag to ignore dependency checks in Emacs25
-    (if (not (configuration-layer//system-package-p p))
-        (package-delete p t t)
-      (message "Would have removed package %s but this is a system package so it has not been changed." pkg-name))))
+  (if-let ((p (car-safe (cdr-safe (assq pkg-name package-alist)))))
```

Comment:
well my bad. I told you to use car-safe + cdr-safe.
But we can use `alist-get` (it's a C subroutine and is very fast).
So
`(if-let ((pkg (alist-get pkg-name package-alist))) ..)`

Reply:
Done.